rubocop.yml

```diff
@@ -8,10 +8,33 @@ inherit_mode:
 
 AllCops:
   SuggestExtensions: false
-  DisabledByDefault: true
   Exclude:
     - "data/**/*"
 
+# All cops except your using extensions are disabled by default.
+Style:
+  Enabled: false
+Layout:
+  Enabled: false
+Lint:
+  Enabled: false
+Metrics:
+  Enabled: false
+Naming:
+  Enabled: false
+Security:
+  Enabled: false
+Bundler:
+  Enabled: false
+Gemspec:
+  Enabled: false
+Performance:
+  Enabled: false
+  Exclude:
+    - "test/**/*"
+Rails:
+  Enabled: false
```

Comment:
As with cop names, these department names should also be listed in ASCII order.

Reply:
Fixed https://github.com/rails/rubocop-rails-omakase/pull/26/commits/b8fb7b1506ff34942852caeed6d8bb0f5f157ba5